core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -434,7 +434,7 @@ Object readResolve() {
      * that JVM runs in, or in the worst case return the default one
      * that's not capable of killing descendants at all.
      */
-    public static ProcessTree get() {
+    public static synchronized ProcessTree get() {
```

Comment:
Fixes `LI_LAZY_INIT_STATIC` for `vetoersExist`. In general, the whole initialization process does not appear to be thread-safe, so synchronizing on the whole method seems safer than doing fine-grained synchronization on `vetoersExist`.